telescope.pri

```diff
@@ -0,0 +1,32 @@
+# This is the basic qmake template for the Ubuntu-SDK
```

Comment:
I'd prefer having the equivalent of that file in a telescope {} config alternative in the src.pro

Reply:
Sure